meinberlin/assets/scss/components/_button.scss

```diff
@@ -21,10 +21,6 @@
 
     &.is-disabled,
     &:disabled {
-        // $is-light: luma($color) > luma($color-contrast);
-        // $weight-bg: if($is-light, 95%, 50%);
-        // $weight-fg: if($is-light, 60%, 10%);
-
         // border-color: mix($color, $color-contrast, $weight-bg);
         // background-color: mix($color, $color-contrast, $weight-bg);
         // color: mix($color, $color-contrast, $weight-fg);
```

Comment:
is this still relevant or can it also be removed?

Reply:
i'm still debating that, I wanted to leave it in till it's been tested a bit in case Caro notices anything amiss and I can update the styles using non planifolia styles